hieradata/common.yaml

```diff
@@ -472,7 +472,9 @@ hosts::production::api::hosts:
   email-campaign-mongo-1:
     ip: '10.1.4.53'
   mapit-1:
-    ip: '10.1.4.60'
+    ip: '10.1.4.60' 
```

Comment:
It looks like there's a whitespace change to this line but I can't see exactly what it is